lib/oneview-sdk/resource/api200/logical_interconnect_group.rb

```diff
@@ -71,15 +71,6 @@ def add_uplink_set(uplink_set)
         @data['uplinkSets'] << uplink_set.data
       end
 
-      # Get the logical interconnect group default settings
-      # @deprecated Use {::get_default_settings} instead. This method will be removed in the next major release.
```

Comment:
We said we'd remove this deprecated method in the next major release, and here it is, so I'm removing it